tests/aws/services/apigateway/test_apigateway_api.py

```diff
@@ -433,6 +433,7 @@ def test_update_rest_api_compression(self, apigw_create_rest_api, snapshot, aws_
         response = aws_client.apigateway.update_rest_api(
             restApiId=api_id, patchOperations=patch_operations
         )
+        # aws_client.apigateway.get_rest_api(restApiId=api_id)
```

Comment:
is this left on purpose? Was there a way to reproduce the issue with the boto client?